roles/archive/templates/etc/cron.weekly/count_archive_hits

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#set -e
+
```

Comment:
I recommend always quoting variables by habit even if the values aren't expected to include whitespace.